src/pin-field.css.ts

```diff
@@ -0,0 +1,92 @@
+import {vars} from './skins/skin-contract.css';
+import {sprinkles} from './sprinkles.css';
+import {keyframes, style} from '@vanilla-extract/css';
+import {desktopFontSize, mobileFontSize} from './text-field-base.css';
+import * as mq from './media-queries.css';
+
+import type {Sprinkles} from './sprinkles.css';
+
+export const disabled = style({
+    opacity: 0.5,
+    cursor: 'default',
+});
+
+const fieldCommonStyles: Sprinkles = {
+    overflow: 'hidden',
+    border: 'regular',
+    display: 'flex',
+    borderRadius: vars.borderRadii.input,
+    position: 'relative',
+    width: 48,
+    height: 48,
+};
+
+export const field = sprinkles({
+    ...fieldCommonStyles,
+    background: vars.colors.backgroundContainer,
+});
+
+export const focusedField = style([
+    sprinkles({
+        ...fieldCommonStyles,
```

Comment:
I think you need to include `background: vars.colors.backgroundContainer,` in this line to avoid the background changes in focus state (only is visible the problem in dark mode)
![image](https://github.com/Telefonica/mistica-web/assets/6722153/68b48f18-1121-4ccc-bb17-ca69545b660f)

Reply:
fixed, good catch!